stable/nginx-ingress/templates/default-backend-deployment.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.defaultBackend.enabled }}
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
Same as previous comment.

> I think this could potentially be a breaking change for some users, so can we just make this configurable with the default being the existing API version?

Reply:
The **Deployment** resource is no longer available in `v1.16` under the `extensions/v1beta1 ` API. It is now only available in the `apps/v1` API. This was available in the `apps/v1` API since `v1.9`. This should be backward compatible as long as `.spec.selector` is explicitly defined within the **Deployment** resource.

Source: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

Reply:
Yeah we shouldn't have any issues since the selector was not defined previously. I'd be comfortable having the default be `apps/v1`, but I would still like to see this be configurable so that users can continue to use an older API if they so desire.

Reply:
done :)